src/platforms/pc/cl_utils.c

```diff
@@ -130,52 +130,65 @@ static void bmp_munmap(struct mmap_data *map)
 {
 	bmp_ident(NULL);
-	DEBUG_WARN("Usage: %s [options]\n", argv[0]);
-	DEBUG_WARN("\t-h\t\t: This help.\n");
-	DEBUG_WARN("\t-v[bitmask]\t: Increasing verbosity. Bitmask:\n");
-	DEBUG_WARN("\t\t\t  1 = INFO, 2 = GDB, 4 = TARGET, 8 = PROBE, 16 = WIRE\n");
-	DEBUG_WARN("\t-l\t\t: List available probes\n");
-	DEBUG_WARN("Probe selection arguments:\n");
-	DEBUG_WARN("\t-d \"path\"\t: Use serial BMP device at <path>");
-#if HOSTED_BMP_ONLY == 1 && defined(__APPLE__)
-	DEBUG_WARN("\n");
-#else
-	DEBUG_WARN(". Deprecated!\n");
-#endif
-	DEBUG_WARN("\t-P <pos>\t: Use debugger found at position <pos>\n");
-	DEBUG_WARN("\t-n <num>\t: Use target device found at position <num>\n");
-	DEBUG_WARN("\t-s \"serial\"\t: Use dongle with (partial) "
-		  "serial number \"serial\"\n");
-	DEBUG_WARN("\t-c \"string\"\t: Use ftdi dongle with type \"string\"\n");
-	DEBUG_WARN("\t\t Use \"list\" to list available cables\n");
-	DEBUG_WARN("Run mode related options:\n");
-	DEBUG_WARN("\tDefault mode is to start the debug server at :2000\n");
-	DEBUG_WARN("\t-j\t\t: Use JTAG. SWD is default.\n");
-	DEBUG_WARN("\t-f\t\t: Set minimum high and low times of SWJ waveform.\n");
-	DEBUG_WARN("\t-C\t\t: Connect under hardware reset\n");
-	DEBUG_WARN("\t-t\t\t: Scan SWD or JTAG and display information about \n"
-			   "\t\t\t  connected devices\n");
-	DEBUG_WARN("\t-T\t\t: Continuous read/write-back some value to allow\n"
-			   "\t\t\t  timing insection of SWJ. Abort with ^C\n");
-	DEBUG_WARN("\t-e\t\t: Assume \"resistor SWD connection\" on FTDI: TDI\n"
-               "\t\t\t  connected to TMS, TDO to TDI with eventual resistor\n");
-	DEBUG_WARN("\t-E\t\t: Erase flash until flash end or for given size\n");
-	DEBUG_WARN("\t-w\t\t: Write binary file to target flash (default).\n");
-	DEBUG_WARN("\t-V\t\t: Verify flash against binary file. Can be combined\n"
-	           "\t\t\t  with -w to verify right after programming.\n");
-	DEBUG_WARN("\t-r\t\t: Read flash and write to binary file\n");
-	DEBUG_WARN("\t-p\t\t: Supplies power to the target (where applicable)\n");
-	DEBUG_WARN("\t-R[h]\t\t: Reset device. Default via SWJ or by hardware(h)\n");
-	DEBUG_WARN("\t-H\t\t: Do not use high level commands (BMP-Remote)\n");
-	DEBUG_WARN("\t-m <target>\t: Use (target)id for SWD multi-drop.\n");
-	DEBUG_WARN("\t-M <string>\t: Run target specific monitor commands. Quote multi\n");
-	DEBUG_WARN("\t\t\t  word strings. Run \"-M help\" for help.\n");
-	DEBUG_WARN("Flash operation modifiers options:\n");
-	DEBUG_WARN("\tDefault action with given file is to write to flash\n");
-	DEBUG_WARN("\t-a <addr>\t: Start flash operation at flash address <addr>\n"
-		"\t\t\t  Default start is start of flash in memory map\n");
-	DEBUG_WARN("\t-S <num>\t: Read <num> bytes. Default is until read fails.\n");
-	DEBUG_WARN("\t <file>\t\t: Use (binary) file <file> for flash operation\n");
+	PRINT_INFO("\n");
+	PRINT_INFO("Usage: %s [-h | -l | [-vBITMASK] [-d PATH | -P NUMBER | -s SERIAL | -c TYPE]\n", argv[0]);
```

Comment:
Do we need to print every single line separately? Would it not be possible to just write:
```
PRINT_INFO(
"my first line of text\n"
"my second line of text\n"
"my third line of text\n");
```
I think this is specified in C89 so it should be fine to use and in my opinion would make this look better and make it slightly easier to edit when needed.

Reply:
We can do it that way, we didn't initially because of the extra preprocessor logic that was in there, and the command line argument, but now that's all sorted, we can do a second pass and fix this.

Reply:
Ohh ok, No worries, I figured that it might be something along those lines. Let's make that additional pass and then we should be good to merge this. :)